ajax/libs/sinon.js/package.json

```diff
@@ -16,6 +16,15 @@
     "type": "git",
     "url": "https://github.com/sinonjs/sinon"
   },
+  "npmName": "sinon",
+  "npmFileMap": [
+    {
+      "basePath": "pkg",
+      "files": [
+        "**/+(sinon.|sinon-no-sourcemaps.)*"
```

Comment:
@fastest963 We don't use "" or "**/" because we don't want all files in pkg. For example, in v1.17.2, there are sinon-ie.js, sinon-ie-1.17.2.js, sinon-ie-1.17.0.js and sinon-ie-1.16.1.js but we only need sinon-ie.js whose filename doesn't include versions, right?